### ISSUE TITLE
Bunpro not loading profile

### ISSUE THREAD

**Issue by DanyRamdas** [ranger-ross/nihongo-stats#528]
Hi,

It seems it is not loading my profile.
Is the integration broken perhaps?

**Comment by ranger-ross** [comment]
Yeah it looks like BunPro removed the APIs Nihongo Stats was using :disappointed: 

I will see if there is a new version to

**Comment by ranger-ross** [comment]
I asked about this in the BunPro forum: https://community.bunpro.jp/t/will-there-ever-be-a-graph-showing-community-stats/61600

Hopefully the BunPro team has another API we can use to replace the current one

**Comment by DanyRamdas** [comment]
Awesome tnx!

Not related to this but is it possible to  integrate https://jpdb.io/ ? They do have api?
As that is what I use instead of anki?
Lastly could I ask for streak dashboard where I see the current streak form these websites?

That would be really awesome, my intended use would be having the website open to see my streak on my extra monitor.

If needed I could and I am willing to send some beers your way! 

Thanks for taking the time to respond.

**Comment by DanyRamdas** [comment]
Hi, me again. 

Did you ever heard back from the devs? Looking at the topic is seems not?

**Comment by ranger-ross** [comment]
> Hi, me again.
> 
> Did you ever heard back from the devs? Looking at the topic is seems not?

Hey, Sorry for the late reply, I have been on vaction.
 I have not heard back from the BP team :disappointed: 


I post a [follow message](https://community.bunpro.jp/t/does-or-will-bunpro-have-a-public-api/205/26?u=sully22) in a different thread on the forum.  (I think posting in the original thread is a bit off topic)

**Comment by DanyRamdas** [comment]
I have left an inquiry asking about the status.
